components/PartForm/index.js

```diff
@@ -10,12 +10,19 @@ export default function PartForm({ onSubmit, formName, defaultData }) {
     const formData = new FormData(event.target);
     const data = Object.fromEntries(formData);
 
-    const allDate = new Date();
-    const day = allDate.getDate();
-    const month = allDate.getMonth() + 1;
-    const year = allDate.getFullYear();
-    const todayDate = day + "." + month + "." + year;
+    // get todays date in Format 'dd.mm.yyyy' IF there is no defaultData (edit-part)
+    let todayDate;
+    if (!defaultData) {
+      const date = new Date();
+      const day = date.getDate();
+      const month = date.getMonth() + 1;
+      const year = date.getFullYear();
+      todayDate = `${day}.${month}.${year}`;
```

Comment:
Instead of this, you could also try using `toLocaleDateString` with a specific option:

```suggestion
      const date = new Date().toLocaleDateString('de-DE', { dateStyle: "medium" })
```
[See documentation on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString)

Reply:
great to see it works in just one line of code 😁👌
it did not come to my mind to look it up

for the id thing: i will keep it for now, bc tomorrow or so i will replace it anyway due to db ✌🏼

Thanks @marcbendixen